en/08_Changelogs/5.4.0.md

```diff
@@ -278,6 +278,10 @@ The exception was made for the following reasons:
 
 ### Deprecated API
 
+The following API has been deprecated in this release, and will be removed or replaced in a future major release.
+
+It's best practice to avoid usnig deprecated code where possible, but sometimes it's unavoidable. This API will all continue to be available at least until the next major release.
```

Comment:
```suggestion
It's best practice to avoid using deprecated code where possible, but sometimes it's unavoidable. This API will all continue to be available at least until the next major release.
```

Reply:
Oops! Fixed